proto/options.proto

```diff
@@ -384,6 +384,11 @@ message Options {
     expansion_type = 6;
   }
 
+  enum ReverseTimeTracking {
+    heuristic = 0;
+    invalid = 1;
```

Comment:
did you do this on purpose to make the default (unset) value be new behavior? once upon a time i would be against this to avoid breaking existing users but honestly i feel like sometimes we have to move forward.

also nit: `invalid` seems a strange word choice, why not `disabled` or `none` or something along those lines